11.PureComponent.md

```diff
@@ -0,0 +1,11 @@
+**Jam3 Lessons - React**
+
+# React.PureComponent
+
+> React.PureComponent is exactly like `React.Component` but implements `shouldComponentUpdate()` with a shallow prop and state comparison.
+
+> If your React component's `render()` function renders the same result given the same props and state, you can use `React.PureComponent` for a performance boost in some cases.
```

Comment:
I think it sounds better with this changes:

_If your component's render function renders the same result given the same props and state, you can use it as a performance boost and write less code._

@Let me know what you think @neo.